containers/azure-terraform-0.12/README.md

```diff
@@ -0,0 +1,39 @@
+# Azure Terraform
```

Comment:
Same comment as in devcontainer.json, given there's two now, this should probably include the version number correct?

Reply:
Fioxed in 30d13b9